Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_description.md

```diff
@@ -0,0 +1 @@
+In order to configure an instance you can configure interval from fetching data from the api.
```

Comment:
@kirbles19 We would love your help here. to configure this integration you do not need creds. just add the refresh interval. all the urls are stored in the integration context and are being refreshed every x time.

Reply:
To configure an integration instance, you only need to set the refresh interval. All of the URLs are stored in the integration context and are refreshed according to the refresh interval.